engine/runtime-integration-tests/src/test/java/org/enso/interpreter/runtime/ManagedResourceTest.java

```diff
@@ -80,7 +83,7 @@ public void regularReference() throws Exception {
     assertFalse("Value was not GCed", getResource.execute(ref).isNull());
     assertEquals("We get the object", weakRef.get(), getResource.execute(ref).asHostObject());
 
-    ensoCtx.getResourceManager().scheduleFinalizationOfSystemReferences();
+    ctxRule.leakContext().getResourceManager().scheduleFinalizationOfSystemReferences();
```

Comment:
Rename to `ensoCtx` or `ensoContext`?
```suggestion
    ctxRule.ensoContext().getResourceManager().scheduleFinalizationOfSystemReferences();
```

Reply:
Renamed in b28a8bee870700ccad72283ad84cd4df9fdba220